tests/test_session.py

```diff
@@ -677,6 +677,50 @@ def test_app_utilities_new_and_old(mixing_elbow_settings_session):
 
 
 @pytest.mark.standalone
+def test_new_launch_fluent_api():
+    import ansys.fluent.core as pyfluent
+
+    solver = pyfluent.Solver.from_install()
+    assert solver.health_check.check_health() == "SERVING"
+
+    ip = solver.connection_properties.ip
+    password = solver.connection_properties.password
+    port = solver.connection_properties.port
+
+    solver_connected = pyfluent.Solver.from_connection(
+        ip=ip, password=password, port=port
```

Comment:
Here we could add a capability to read the server-info file directly as we do in fluent-connection.

Reply:
We can pass server-info file here.

Reply:
Just noticed that i already exists. Please ignore.